src/LibMtpSharpStandardMacOS/LibMtpSharpStandardMacOS.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>9</LangVersion>
-    <Version>0.1.6</Version>
+    <Version>0.1.8</Version>

```

Comment:
We can use next number 0.1.7

Reply:
I added 0.1.8 version because 0.1.7 has bug in code (I detected this bug after publishing my 0.1.7 version)